src/MongoDB/Monitoring/CommandFailedEvent.c

```diff
@@ -1,24 +1,18 @@
 /*
-  +---------------------------------------------------------------------------+
-  | PHP Driver for MongoDB                                                    |
-  +---------------------------------------------------------------------------+
-  | Copyright 2016-2016 MongoDB, Inc.                                         |
-  |                                                                           |
-  | Licensed under the Apache License, Version 2.0 (the "License");           |
-  | you may not use this file except in compliance with the License.          |
-  | You may obtain a copy of the License at                                   |
-  |                                                                           |
-  | http://www.apache.org/licenses/LICENSE-2.0                                |
-  |                                                                           |
-  | Unless required by applicable law or agreed to in writing, software       |
-  | distributed under the License is distributed on an "AS IS" BASIS,         |
-  | WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  |
-  | See the License for the specific language governing permissions and       |
-  | limitations under the License.                                            |
-  +---------------------------------------------------------------------------+
-  | Copyright (c) 2016-2016 MongoDB, Inc.                                     |
-  +---------------------------------------------------------------------------+
-*/
+ * Copyright 2017 MongoDB, Inc.
```

Comment:
I didn't realize the code was written so long ago, but https://github.com/mongodb/mongo-php-driver/pull/466 dates back to November 2016. Will update.